packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -98,17 +99,27 @@ const responseStreamActor = async (callback: StreamActorCallback) => {
     }
   } catch (error: unknown) {
     let returnedError = error;
+
+    const isConnectionTimeout = isConnectionTimeoutError(error);
+
     if (isInvalidSignatureRegionException(error)) {
       returnedError = new Error(
         'Invalid region in FaceLivenessDetector or credentials are scoped to the wrong region.'
       );
     }
 
     if (returnedError instanceof Error) {
-      callback({
-        type: 'SERVER_ERROR',
-        data: { error: returnedError },
-      });
+      if (isConnectionTimeout) {
```

Comment:
nit, since this is the only place we use the `isConnectionTimeout` const, could probably just call the function?
```suggestion
      if (isConnectionTimeoutError(error)) {
```

Reply:
Thanks, updated!